docs/extensions/submit-extension.md

```diff
@@ -14,51 +14,32 @@ To ease the review, you have to open a specific PR for each deployment request w
 
 <h6>Table of Contents</h6>
 
--   [Create your Pull Request](#create-your-pull-request)
-  -   [Pull Request labels](#pull-request-labels)
-  -   [Approval and rejection](#approval-and-rejection)
+-   [Submit your extension](#create-your-pull-request)
+-   [Pull Request labels](#pull-request-labels)
+-   [Approval and rejection](#approval-and-rejection)
 -   [Extension deployment](#extension-deployment)
-  -   [Multiple versions](#multiple-versions)
+-   [Multiple versions](#multiple-versions)
 
-## Create your Pull Request
+## Submit your extension
 
-You must use the Pull Request template provided by LumApps. This template is available in the `pull_request_template.md` file in a `.github` folder.
-This folder has to be in the root folder of your repository so GitHub can find it and automatically apply the template to your PR.
-
-You must fill in the template to give our code reviewers enough information about the feature or fix you want to deploy.
-
-### Pull Request labels
-
-We use labels on PR to give developers visibility on review and deployment workflow.
-
-| Label       | Description                                                                                                                  |
-| ----------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| need review | Ask for a code review. LumApps performs the code review.                                                                     |
-| validated   | LumApps has reviewed and validated your extension.                                                                           |
-| rejected    | LumApps has reviewed and rejected your extension.                                                                            |
-| need deploy | Ask for a production deployment of your extension. The deployment is performed by LumApps **after** a validated code review. |
-| deployed    | LumApps has deployed your extension. Customers can now use it in production.                                                 |
+The extension submission is done from the developer portal.
 
 ### Approval and rejection
 
 After an extension version is reviewed by LumApps, there are two possible outcomes:
 
--   **Approved** means your extension respects our guidelines. You can ask for production deployment for your extension. See [Extension Deployment](/submit-extension.md/extension_deployment).
--   **Rejected** means that your extension may not respect our guidelines and some revisions are required. The reason is provided in the PR. You can edit your extension, test it again, and resubmit your extension when you are ready.
+-   **Approved** means your extension respects our guidelines and you are allowed to deploy it.
```

Comment:
Once approved, the extension is automatically deployed

Reply:
done